Description: Allow for preceding newlines when checking for error message on undecodable argv.
Origin: commit, revision id: jelmer@samba.org-20110601121444-zzggg2lyhdhp03uw
Author: Jelmer Vernooij <jelmer@samba.org>
Last-Update: 2011-06-01
Applied-Upstream: no
X-Bzr-Revision-Id: jelmer@samba.org-20110601121444-zzggg2lyhdhp03uw

=== modified file 'bzrlib/tests/blackbox/test_exceptions.py'
--- old/bzrlib/tests/blackbox/test_exceptions.py	2011-05-26 08:05:45 +0000
+++ new/bzrlib/tests/blackbox/test_exceptions.py	2011-06-01 12:14:44 +0000
@@ -58,7 +58,7 @@
             env_changes={"LANG": "C", "LC_ALL": "C"},
             universal_newlines=True,
             retcode=errors.EXIT_ERROR)
-        self.assertContainsRe(err, r"^bzr: ERROR: .*'\\xa0'.* unsupported")
+        self.assertContainsRe(err, r"bzr: ERROR: .*'\\xa0'.* unsupported")
         self.assertEquals(out, "")
 
 

